### PR TITLE
Just using the newer Versions of some gradle Plugins

### PR DESCRIPTION
**Pull request by JuliusKreutz**
The only important changes are in build.gradle.kts . But I also fixed some typos :penguin: Build fine on my device with IntelliJ 2020.3.2 but shoud also work on any other IntelliJ 2020.3 version.

### CODE REVIEW

File: src/main/resources/i18n/OrionBundle.properties [hunk 1]
```diff
@@ -1,18 +1,17 @@
 orion.settings.displayName=Orion
 orion.settings.url.title=Artemis base URL:
-orion.settings.url.label=The URL of the ArTEMiS homepage.\nChange this to change between different versions of Artemis
+orion.settings.url.label=The URL of the ArTEMiS homepage.\nChange this to change between different versions of Artemis.
```
Comment (krusche) [inline src/main/resources/i18n/OrionBundle.properties:3]:
```suggestion
orion.settings.url.label=The URL of the Artemis homepage.\nChange this to change between different versions of Artemis.
```

File: src/main/java/de/tum/www1/orion/vcs/OrionGitAdapter.kt [hunk 2]
```diff
@@ -258,11 +258,11 @@ object OrionGitAdapter {
 
     private fun resetAndPull(module: Module) {
         ProgressManager.getInstance().run(object :
-            Task.Modal(module.project, "Updating your exercise files...", false) {
+            Task.Modal(module.project, "Updating Your Exercise Files...", false) {
```
Comment (krusche) [inline src/main/java/de/tum/www1/orion/vcs/OrionGitAdapter.kt:261]: I am not a big fan of capital case in normal sentences, I would leave it as it was

**Comment by krusche** [comment]
thanks for the contribution

File: src/main/java/de/tum/www1/orion/vcs/OrionGitAdapter.kt (hunk 2 above)
Comment (JuliusKreutz) [reply src/main/java/de/tum/www1/orion/vcs/OrionGitAdapter.kt]: Fixed! :)

**Pull request merged**